runtimes/rust/src/error.rs

```diff
@@ -21,7 +21,7 @@ use exonum_derive::ExecutionFail;
 #[derive(ExecutionFail)]
 #[execution_fail(kind = "runtime")]
 pub enum Error {
-    /// Unable to parse artifact identifier or specified artifact has non-empty spec.
+    /// Cannot deploy artifact because it has non-empty specification.
```

Comment:
```suggestion
    /// Cannot deploy an artifact because it has non-empty specification.
```

Reply:
The error concerns a specific artifact though. IMO, dropping an article in an error message is fine.